src/EditorFeatures/Core/Implementation/Suggestions/FixMultipleOccurrencesService.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Service to compute and apply <see cref="FixMultipleCodeAction"/> code fixes.
+    /// </summary>
+    [ExportWorkspaceServiceFactory(typeof(IFixMultipleOccurrencesService), ServiceLayer.Host), Shared]
```

Comment:
@heejaechang I just took a closer look and the reason they are separate is that this new service needs the edit handler, which is only defined at the EditorFeatures layer, and the CodeFixService is defined at the Features layer.